README.md

```diff
@@ -38,13 +44,13 @@ Bulk delivery methods (one notification for all recipients):
 [Watch Screencast](https://www.youtube.com/watch?v=Scffi4otlFc)
 
 ## 🚀 Installation
-Run the following command to add Noticed to your Gemfile:
+Run the following command to add Noticed to your Gemfile
 
 ```ruby
 bundle add "noticed"
 ```
 
-Add the migrations:
+Add the migraitons
```

Comment:
```suggestion
Add the migrations
```

Reply:
D'oh! Thought I caught that one. Thanks for the suggestion! 👍